base/random.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <random>
+#include <vector>
+
+namespace my
```

Comment:
We decided to use namespace base in the base subdirectory, instead of my. What do you think?